mmdeploy/python/adaptive_avg_pool_ncnn.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+
+class NcnnAdaptiveAvgPoolingOp(torch.autograd.Function):
```

Comment:
Errr, this is not what I means, there is an symbolic rewrite of adaptive_avg_pool here https://github.com/open-mmlab/mmdeploy/blob/master/mmdeploy/pytorch/ops/adaptive_avg_pool.py . You can add another rewrite for ncnn.

Reply:
> Errr, this is not what I means, there is an symbolic rewrite of adaptive_avg_pool here https://github.com/open-mmlab/mmdeploy/blob/master/mmdeploy/pytorch/ops/adaptive_avg_pool.py . You can add another rewrite for ncnn.

Ah, got it.